config/packages/sentry.yaml

```diff
@@ -0,0 +1,20 @@
+when@prod:
+    sentry:
+        dsn: "https://c70286fb157048be9ebc6e918e8c2b79@o200403.ingest.sentry.io/1323198"
```

Comment:
I think we should keep the `env(SENTRY_DSN):` is this *can* be overloaded on an implementation level, even if mostly we just stick with the default we provide.

Reply:
:heavy_check_mark: